Dockerfile

```diff
@@ -2,99 +2,97 @@
 # It also expects to be contained in the /navigation2 root folder for file copy
 # Example build command:
 # sudo docker build -t nav2:latest --build-arg http_proxy=http://my.proxy.com:### .
-FROM osrf/ros2:bouncy-desktop
-
-SHELL ["/bin/bash", "-c"]
+FROM osrf/ros2:nightly
 
 # setup keys
 # check if proxy is set and get keys, using proxy if it is set
-RUN if [ "$http_proxy" == "" ]; \
+RUN if [ "$http_proxy" != "" ]; \
```

Comment:
You got rid of setting this key when there is no proxy variable set. Seems we shouldn't need it when there is a proxy either. Any reason you left it here?

Reply:
I added back a build arg in 5b460ff